src/custom_nodes/face_blur/Dockerfile.ubuntu

```diff
@@ -16,20 +16,40 @@
 
 FROM ubuntu:20.04
 RUN apt update && apt install -y build-essential curl
-ARG OPENCV_DOWNLOAD_FOLDER
-RUN curl -s ${OPENCV_DOWNLOAD_FOLDER}/openvino_opencv_ubuntu20.tgz | tar -xzf -
 ARG NODE_NAME=face_blur
 
 ARG OPS="-fpic -O2 -U_FORTIFY_SOURCE -fstack-protector -fno-omit-frame-pointer -D_FORTIFY_SOURCE=1 -fno-strict-overflow -Wall -Wno-unknown-pragmas -Werror -Wno-error=sign-compare -fno-delete-null-pointer-checks -fwrapv"
 
+# Build OpenCV from source
+RUN apt-get update && \
+    DEBIAN_FRONTEND=noninteractive \
+    apt-get install -y --no-install-recommends \
+        git \
+        cmake \
+    && rm -rf /var/lib/apt/lists/*
+ARG OPENCV_BRANCH="4.6.0"
+
+WORKDIR /opt
+RUN git clone https://github.com/opencv/opencv.git --depth 1 -b ${OPENCV_BRANCH}
+
+COPY opencv_cmake_flags.txt /opt/opencv
+
+WORKDIR /opt/opencv/build
+# hadolint ignore=SC2046
+RUN cmake $(cat /opt/opencv/opencv_cmake_flags.txt) /opt/opencv && \
+    make "-j$(nproc)" && \
+    make install
+# End of OpenCV building
+
+WORKDIR /
```

Comment:
Right now we have basically copy-paste mostly the same steps in several Dockerfiles. Wouldn't it be better to have multi stage build so that we require to build OpenCV once per base OS and then reuse this build across OVMS/all CN's?

Reply:
Agreed to separate new tasks and put to next sprint